From dd6b9d94eed7c3e5bebf4f92b3acdcbe31705959 Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Sun, 12 Apr 2020 00:33:14 +0300
Subject: [PATCH 123/133] fix my complex derp

	modified:   packages/SystemUI/res/color/qs_background_dark.xml
	modified:   packages/SystemUI/res/drawable/qs_background_primary.xml
	modified:   packages/SystemUI/res/layout/qs_panel.xml
	modified:   packages/overlays/Android.mk
	modified:   packages/overlays/CharcoalDarkUIOverlay/AndroidManifest.xml
	modified:   packages/overlays/CharcoalDarkUIOverlay/res/values-night/colors.xml
	modified:   packages/overlays/MidnightDarkUIOverlay/AndroidManifest.xml
	modified:   packages/overlays/MidnightDarkUIOverlay/res/values-night/colors.xml
---
 .../SystemUI/res/color/qs_background_dark.xml |  4 +-
 .../res/drawable/qs_background_primary.xml    |  2 +-
 packages/SystemUI/res/layout/qs_panel.xml     |  2 +-
 packages/overlays/Android.mk                  |  2 +-
 .../CharcoalDarkUIOverlay/AndroidManifest.xml |  2 +-
 .../res/values-night/colors.xml               | 39 +++++++++---------
 .../MidnightDarkUIOverlay/AndroidManifest.xml |  2 +-
 .../res/values-night/colors.xml               | 41 +++++++++----------
 8 files changed, 46 insertions(+), 48 deletions(-)

diff --git a/packages/SystemUI/res/color/qs_background_dark.xml b/packages/SystemUI/res/color/qs_background_dark.xml
index aa05cd8fa03..c19fa080324 100644
--- a/packages/SystemUI/res/color/qs_background_dark.xml
+++ b/packages/SystemUI/res/color/qs_background_dark.xml
@@ -15,6 +15,6 @@
 -->
 
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
-    <item android:alpha="1"
-          android:color="@android:color/primary_device_default_settings"/>
+    <item android:alpha="0.87"
+          android:color="?android:attr/colorBackgroundFloating"/>
 </selector>
diff --git a/packages/SystemUI/res/drawable/qs_background_primary.xml b/packages/SystemUI/res/drawable/qs_background_primary.xml
index e501224db3c..dd74cadd095 100644
--- a/packages/SystemUI/res/drawable/qs_background_primary.xml
+++ b/packages/SystemUI/res/drawable/qs_background_primary.xml
@@ -15,7 +15,7 @@
 -->
 <inset xmlns:android="http://schemas.android.com/apk/res/android">
     <shape>
-        <solid android:color="@android:color/primary_device_default_settings"/>
+        <solid android:color="@color/qs_background_dark"/>
         <corners android:radius="?android:attr/dialogCornerRadius" />
     </shape>
 </inset>
diff --git a/packages/SystemUI/res/layout/qs_panel.xml b/packages/SystemUI/res/layout/qs_panel.xml
index e81c2da74bb..01dfeb281e1 100644
--- a/packages/SystemUI/res/layout/qs_panel.xml
+++ b/packages/SystemUI/res/layout/qs_panel.xml
@@ -36,7 +36,7 @@
         android:layout_height="@*android:dimen/quick_qs_offset_height"
         android:clipToPadding="false"
         android:clipChildren="false"
-        android:background="#f0000000" />
+        android:background="#ff000000" />
 
     <!-- Gradient view behind QS -->
     <View
diff --git a/packages/overlays/Android.mk b/packages/overlays/Android.mk
index 3c9367d104c..50617297575 100644
--- a/packages/overlays/Android.mk
+++ b/packages/overlays/Android.mk
@@ -81,7 +81,7 @@ LOCAL_REQUIRED_MODULES := \
 	QSPanel34Overlay \
 	QSPanel43Overlay \
 	QSPanel55Overlay \
-	CharcoalBlackOverlay \
+	CharcoalDarkUIOverlay \
 	MidnightDarkUIOverlay \
 	NavigationBarMode3ButtonOverlay \
 	NavigationBarMode2ButtonOverlay \
diff --git a/packages/overlays/CharcoalDarkUIOverlay/AndroidManifest.xml b/packages/overlays/CharcoalDarkUIOverlay/AndroidManifest.xml
index d03ba26d5b2..fa3878ba73d 100644
--- a/packages/overlays/CharcoalDarkUIOverlay/AndroidManifest.xml
+++ b/packages/overlays/CharcoalDarkUIOverlay/AndroidManifest.xml
@@ -20,5 +20,5 @@
     android:versionCode="1"
     android:versionName="1.0">
     <overlay android:targetPackage="android" android:category="android.theme.customization.darkui" android:priority="1"/>
-    <application android:label="CharcoalBlack" android:hasCode="false"/>
+    <application android:label="Charcoal Black" android:hasCode="false"/>
 </manifest>
diff --git a/packages/overlays/CharcoalDarkUIOverlay/res/values-night/colors.xml b/packages/overlays/CharcoalDarkUIOverlay/res/values-night/colors.xml
index 7360755c630..a523cee654d 100644
--- a/packages/overlays/CharcoalDarkUIOverlay/res/values-night/colors.xml
+++ b/packages/overlays/CharcoalDarkUIOverlay/res/values-night/colors.xml
@@ -17,26 +17,25 @@
  */
 -->
 <resources>
-    <color name="notification_primary_text_color_dark">#f0e5e5e5</color>
+    <color name="notification_primary_text_color_dark">#ffe5e5e5</color>
     <color name="notification_secondary_text_color_dark">#b3e5e5e5</color>
-    <color name="notification_default_color_dark">#f0e5e5e5</color>
-    <color name="notification_material_background_color">#f00f1012</color> <!-- material_grey_900 -->
-    <color name="primary_material_dark">#f00f1012</color> <!-- material_grey_900 -->
-    <color name="primary_material_settings">#f00f1012</color> <!-- material_grey_900 -->
+    <color name="notification_default_color_dark">#ffe5e5e5</color>
+    <color name="notification_material_background_color">#ff0f1012</color> <!-- material_grey_900 -->
+    <color name="primary_material_dark">#ff0f1012</color> <!-- material_grey_900 -->
+    <color name="primary_material_settings">#ff0f1012</color> <!-- material_grey_900 -->
     <color name="primary_dark_material_dark">@android:color/black</color>
-    <color name="primary_dark_material_settings">#f00f1012</color> <!-- material_grey_850 -->
-    <color name="primary_dark_device_default_dark">#f00f1012</color> <!-- material_grey_850 -->
-    <color name="primary_dark_device_default_settings">#f00f1012</color> <!-- material_grey_850 -->
-    <color name="secondary_material_settings">#f01a1b1e</color> <!-- material_grey_800 -->
-    <color name="tertiary_material_settings">#f0202126</color>
-    <color name="quaternary_material_settings">#f0dadce0</color> <!-- material_grey_200 -->
-		<color name="notification_material_background_color">#f00f1012</color>
-    <color name="material_grey_900">#f00f1012</color>
-    <color name="material_grey_850">#f00f1012</color>
-    <color name="material_grey_800">#f01a1b1e</color>
-    <color name="material_grey_600">#f0202126</color>
-    <color name="material_grey_300">#f03c4043</color>
-    <color name="material_grey_200">#f0dadce0</color>
-    <color name="material_grey_100">#f0eeeeee</color>
-    <color name="material_grey_50">#f0e8eaed</color>
+    <color name="primary_dark_material_settings">#ff0f1012</color> <!-- material_grey_850 -->
+    <color name="primary_dark_device_default_dark">#ff0f1012</color> <!-- material_grey_850 -->
+    <color name="primary_dark_device_default_settings">#ff0f1012</color> <!-- material_grey_850 -->
+    <color name="secondary_material_settings">#ff1a1b1e</color> <!-- material_grey_800 -->
+    <color name="tertiary_material_settings">#ff202126</color>
+    <color name="quaternary_material_settings">#ffdadce0</color> <!-- material_grey_200 -->
+    <color name="material_grey_900">#ff0f1012</color>
+    <color name="material_grey_850">#ff0f1012</color>
+    <color name="material_grey_800">#ff1a1b1e</color>
+    <color name="material_grey_600">#ff202126</color>
+    <color name="material_grey_300">#ff3c4043</color>
+    <color name="material_grey_200">#ffdadce0</color>
+    <color name="material_grey_100">#ffeeeeee</color>
+    <color name="material_grey_50">#ffe8eaed</color>
 </resources>
diff --git a/packages/overlays/MidnightDarkUIOverlay/AndroidManifest.xml b/packages/overlays/MidnightDarkUIOverlay/AndroidManifest.xml
index b7528ef1fb2..84d8029a0e1 100644
--- a/packages/overlays/MidnightDarkUIOverlay/AndroidManifest.xml
+++ b/packages/overlays/MidnightDarkUIOverlay/AndroidManifest.xml
@@ -20,5 +20,5 @@
     android:versionCode="1"
     android:versionName="1.0">
     <overlay android:targetPackage="android" android:category="android.theme.customization.darkui" android:priority="1"/>
-    <application android:label="Midnight" android:hasCode="false"/>
+    <application android:label="Midnight Blue" android:hasCode="false"/>
 </manifest>
diff --git a/packages/overlays/MidnightDarkUIOverlay/res/values-night/colors.xml b/packages/overlays/MidnightDarkUIOverlay/res/values-night/colors.xml
index 1f2c7879e57..de50bd007f8 100644
--- a/packages/overlays/MidnightDarkUIOverlay/res/values-night/colors.xml
+++ b/packages/overlays/MidnightDarkUIOverlay/res/values-night/colors.xml
@@ -17,26 +17,25 @@
  */
 -->
 <resources>
-    <color name="notification_primary_text_color_dark">#f0e7faff</color>
+    <color name="notification_primary_text_color_dark">#ffe7faff</color>
     <color name="notification_secondary_text_color_dark">#b3e7faff</color>
-    <color name="notification_default_color_dark">#f0e7faff</color>
-    <color name="notification_material_background_color">#f00d2029</color> <!-- material_grey_900 -->
-    <color name="primary_material_dark">#f00d2029</color> <!-- material_grey_900 -->
-    <color name="primary_material_settings">#f00d2029</color> <!-- material_grey_900 -->
-    <color name="primary_dark_material_dark">#f009181e</color>
-    <color name="primary_dark_material_settings">#f00d2029</color> <!-- material_grey_850 -->
-    <color name="primary_dark_device_default_dark">#f00d2029</color> <!-- material_grey_850 -->
-    <color name="primary_dark_device_default_settings">#f00d2029</color> <!-- material_grey_850 -->
-    <color name="secondary_material_settings">#f01b2e38</color> <!-- material_grey_800 -->
-    <color name="tertiary_material_settings">#f01e343f</color>
-    <color name="quaternary_material_settings">#f03c4043</color> <!-- material_grey_200 -->
-    <color name="notification_material_background_color">#f00d2029</color>
-    <color name="material_grey_900">#f00d2029</color>
-    <color name="material_grey_850">#f00d2029</color>
-    <color name="material_grey_800">#f01b2e38</color>
-    <color name="material_grey_600">#f01e343f</color>
-    <color name="material_grey_300">#f09db3be</color>
-    <color name="material_grey_200">#f09aadb7</color>
-    <color name="material_grey_100">#f0cde0ea</color>
-    <color name="material_grey_50">#f0e7faff</color>
+    <color name="notification_default_color_dark">#ffe7faff</color>
+    <color name="notification_material_background_color">#ff0d2029</color> <!-- material_grey_900 -->
+    <color name="primary_material_dark">#ff0d2029</color> <!-- material_grey_900 -->
+    <color name="primary_material_settings">#ff0d2029</color> <!-- material_grey_900 -->
+    <color name="primary_dark_material_dark">#ff09181e</color>
+    <color name="primary_dark_material_settings">#ff0d2029</color> <!-- material_grey_850 -->
+    <color name="primary_dark_device_default_dark">#ff0d2029</color> <!-- material_grey_850 -->
+    <color name="primary_dark_device_default_settings">#ff0d2029</color> <!-- material_grey_850 -->
+    <color name="secondary_material_settings">#ff1b2e38</color> <!-- material_grey_800 -->
+    <color name="tertiary_material_settings">#ff1e343f</color>
+    <color name="quaternary_material_settings">#ff3c4043</color> <!-- material_grey_200 -->
+    <color name="material_grey_900">#ff0d2029</color>
+    <color name="material_grey_850">#ff0d2029</color>
+    <color name="material_grey_800">#ff1b2e38</color>
+    <color name="material_grey_600">#ff1e343f</color>
+    <color name="material_grey_300">#ff9db3be</color>
+    <color name="material_grey_200">#ff9aadb7</color>
+    <color name="material_grey_100">#ffcde0ea</color>
+    <color name="material_grey_50">#ffe7faff</color>
 </resources>
-- 
2.17.1

